docs/src/main/paradox/routing-dsl/source-streaming-support.md

```diff
@@ -17,26 +17,23 @@ objects as a continuous HTTP request or response. The elements are most often se
 however do not have to be. Concatenating elements side-by-side or emitting "very long" JSON array is also another
 use case.
 
-In the below examples, we'll be referring to the `Tweet` and `Measurement` case classes as our model, which are defined as:
+In the below examples, we'll be referring to the `Tweet` case class as our model, which is defined as:
```

Comment:
As in the description of the pull request, removing the mention to `Measurement`, as it is not used immediately here.